config/locales/en/casework.yml

```diff
@@ -38,6 +38,16 @@ en:
       cannot_send_back_when_marked_as_ready: This application was already marked as ready for assessment
       cannot_download_doc_uploaded_to_another_app: File must be uploaded to current application to download
       cannot_download_try_again: "%{file_name} could not be downloaded – try again"
+      cannot_view_apps_without_a_work_stream:
+        - You must be allocated to a work queue to view applications
```

Comment:
Probably need to change work queue but type of work sounds awkward here

Reply:
Agree